imager/ImagerProfile/models.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import unicode_literals
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils.encoding import python_2_unicode_compatible
+
+
+class ActiveImagerManager(models.Manager):
+    def get_queryset(self):
+        return super(ActiveImagerManager,
+                     self).get_queryset().filter(user__is_active=True)
+
+
+@python_2_unicode_compatible
+class ImagerProfile(models.Model):
+    """Imager profile that is attached to user on creation"""
+    user = models.OneToOneField(User, related_name='profile')
```

Comment:
Be consistent about putting empty lines between field declarations or not.  The "django way" is to not do so.